src/database/config/config.js

```diff
@@ -9,18 +9,18 @@ module.exports = {
     host: process.env.DB_HOST,
     dialect: 'postgres',
     dialectOptions: {
-    ssl: {"require":true,
-    rejectUnauthorized: false,
+      ssl: { "require":true,
+      rejectUnauthorized: false,
```

Comment:
ssl is not needed during development. It is essential in production.

Reply:
It was since we were connecting to a hosted testDB.